tests/cpp.test

```diff
@@ -793,3 +793,4 @@
 60045  align_asterisk_after_type_cast.cfg   cpp/align_asterisk_after_type_cast.cpp
 60046  align_continuation_left_shift.cfg    cpp/align_continuation_left_shift.cpp
 60047  align_default_after_override.cfg     cpp/align_default_after_override.cpp
+60048  2322.cfg                             cpp/2322.cpp
```

Comment:
BTW, we have very few test cases named like this, and it isn't necessarily obvious what this means. Can you please use the more common "bug_2322" (or "issue_2322") instead?

Reply:
Done.

BTW, should corresponding statement be added to [CONTRIBUTING.md](https://github.com/uncrustify/uncrustify/blob/master/CONTRIBUTING.md)?

Reply:
Good question. Done in #2408; thanks! :slightly_smiling_face: